en/codelists-guides/codelist-management.rst

```diff
@@ -2,108 +2,115 @@ IATI Codelist Management
+Codes in Replicated Codelists may be:
 
-Once a change has been agreed, a GitHub issue will be created and will be linked to from the forum post.
+- Added - at any point in time, subject to applicable notification by the third-party
+- Modified - at any point in time, subject to applicable notification by the third-party
+- Withdrawn - at any point in time, subject to applicable notification by the third-party
+- Removed - never
 
-When the change has been made and accepted into the master branch of the `Non-embedded Codelist repository <https://github.com/IATI/IATI-Codelists-NonEmbedded>`__, the change will be 'live' to anyone who consumes these codelists directly from Github or via the API. 
+Examples of Replicated Codelists include: DAC 5-Digit, Organisation Registration Agency
 
-At this point the technical team will:
+Non-Embedded Codelists - External
+----------------------
 
- * Update the `Non-Embedded Codelist changelog record <http://iatistandard.org/upgrades/nonembedded-codelist-changelog>`__
- * Regenerate the IATI Standard Website.
- * Mark the forum post as 'Done'
+External Codelists are a collection of codes managed by a third-party organisation. The IATI Technical Team does not maintain a correspoding copy in a document that conforms to the Codelist Schema.
 
-A :doc:`summary of all changes to all non-emebedded </upgrades/nonembedded-codelist-changelog/>` codelists is published on the IATI Standard website. They can also be found via the list of commits to the GitHub repository.
+**Modification Schedule: Non-Embedded Codelists - External**
 
-Special Cases
-*************
+Codes in External Codelists may be:
 
-File Format
-  The IATI Technical Team will use its judgement to assess the importance of a change to the file format list and whether or not an announcement is required (i.e. the list may alter to stay inline with the external source, but announcements will not necessarily be made).
+- Added - at any point in time
+- Modified - at any point in time
+- Withdrawn - at any point in time
+- Removed - at any point in time
+
+Examples of Exteral Codelists include: Earmarking Modalities
+
+Changing Codelist Type
+----------------------
+
+A Codelist may have its type changed. The points at which this may occur are defined by the following modification schedule:
+
+**Modification Schedule: Codelist Type**
+
+A Codelist may have its type changed through the applicable consultation process via the following steps:
+
+- External to Replicated - at a Minor or Major upgrade
+- Replicated to Non-Core - at a Minor or Major upgrade
+- Non-Core to Core - at a Minor or Major upgrade
+- Core to Non-core - at a Minor or Major upgrade
+- Non-Core to Replicated - at a Minor or Major upgrade
+- Replicated to External - at a Minor or Major upgrade
```

Comment:
This is quite a confusing list. Is it sufficient to scrap the list and simply say that codelist type can be changed at a version upgrade, subject to consultation?

Reply:
We've re-looked at this process. They shouldn't all have the same process for changing, due to the significance that the change requires. Have a look at: https://github.com/IATI/IATI-Extra-Documentation/pull/552